File: src/main/kotlin/com/chopeks/psi/SquirrelElementType.java
package com.chopeks.psi;

import com.chopeks.SquirrelLanguage;
import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class SquirrelElementType extends IElementType {
    public SquirrelElementType(@NotNull @NonNls String debugName) {
        super(debugName, SquirrelLanguage.INSTANCE);
    }
}

File: src/main/kotlin/com/chopeks/psi/SquirrelElementType.kt
package com.chopeks.psi

import com.chopeks.SquirrelLanguage
import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls

class SquirrelElementType(debugName: @NonNls String) : IElementType(debugName, SquirrelLanguage)